### PR TITLE
{devel}[GNU/4.9.3-2.25] CMake 3.3.2 + ncurses 6.0 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
some overlap with #1976, but I'm sure @wpoely86 can handle it ;)

File: easybuild/easyconfigs/n/ncurses/ncurses-6.0-GNU-4.9.3-2.25.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+configopts = [
+    # default build
+    '--with-shared --enable-overwrite',
+    # the UTF-8 enabled version (ncursesw)
+    '--with-shared --enable-overwrite --enable-ext-colors --enable-widec --includedir=%(installdir)s/include/ncursesw/'
+]
+
+libs = ["form", "menu", "ncurses", "panel"]
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp", "infotocap", "ncurses5-config",
```
Comment (wpoely86) [inline easybuild/easyconfigs/n/ncurses/ncurses-6.0-GNU-4.9.3-2.25.eb]: `ncurses%(version_major)s-config`

Copy the files from #1977 as they have been tested?

Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-6.0-GNU-4.9.3-2.25.eb]: ok, fine :P

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0db6fc01e289d4ca6e28 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/84c5d6d45dea1ea2f910 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a3bf1ebc908ec7c27fc8 for a full test report.

**Comment by boegel** [comment]
Thanks for the review/test @wpoely86!

**Pull request merged**